### PR TITLE
fix typos in README.md

### PR DESCRIPTION
**Pull request by rahul0x00**
Issue #

### Changes Proposed

I found a few typos, these are minor suggestions to improve formatting and readability.

### Explanation of Changes

1. In the "Options" section, there's a small formatting issue with the last line of the usage text:
    Original: "-i, --info Info displays basic info of the scanned site` </pre>"
    Suggested Change: "-i, --info Info displays basic info of the scanned site</pre>"

2. In the "Installation" section, there's a minor grammatical issue in the sentence: "The script install.sh will first install the latest version of torbot found in pypi,". You can make it more grammatically correct like this:
    Suggested Change: "The script install.sh will first install the latest version of torbot found on PyPI,"

3.  In the "Using Docker" section, there's a missing word in this sentence: "Ensure than you have a tor container running on port 9050." It should be "Ensure that you have a tor container running on port 9050."

4. In the "References" section, there are some minor formatting issues with the citation numbers. For consistency, you can format them like this:
    Original: "1. M. Glassman and M. J. Kang, “Intelligence in the internet age..."
    Suggested Change: "1. M. Glassman and M. J. Kang, “Intelligence in the Internet age..."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ We welcome contributions to this project! Here are a few guidelines to follow:
 
 ## References
 
-    1.  M. Glassman and M. J. Kang, “Intelligence in the internet age: The emergence and evolution of Open Source Intelligence (OSINT),” Comput. Human Behav., vol. 28, no. 2, pp. 673–682, 2012.
+    1.  M. Glassman and M. J. Kang, “Intelligence in the Internet age: The emergence and evolution of Open Source Intelligence (OSINT),” Comput. Human Behav., vol. 28, no. 2, pp. 673–682, 2012.
```
Comment (KingAkeem) [inline README.md:172]: Internet isn't capitalized in the book title. https://www.sciencedirect.com/science/article/abs/pii/S0747563211002585

**Review by KingAkeem — CHANGES_REQUESTED** [review]
Everything looks good besides line 172. I'll approve and merge once that's updated.

**Review by rahul0x00 — COMMENTED** [review]
Resolved

**Review by KingAkeem — APPROVED** [review]
Great work and thanks for the contribution

**Pull request merged**